### ISSUE TITLE
Hover effects broken after prolonged use

### ISSUE THREAD

**Issue by shammellee** [philc/vimium#3478]
After prolonged browsing, hover effects (eg, links with underlines) stop working. Hover effects work again after disabling then reenabling Vimium.

chrome version: 79.0.3945.117
vimium version: 1.64.6

**Comment by gdh1995** [comment]
Could you provide the URLs of the buggy web pages and your operating system's brand and version?



---Original---

**Comment by shammellee** [comment]
@gdh1995 updated 👆

**Comment by gdh1995** [comment]
Will the page get slower and slower when this issue happens?

This is a little strange - on amazon.com, its links of goods will cause page reloading, so I think there should have been no memory or event listener leaks.

**Comment by shammellee** [comment]
@gdh1995 no, page speed/performance is fine. I'm on Amazon a lot, so that's just where I notice it. Though it has happened on other sites as well. As I said, hover effects work again after disabling the extension.

**Comment by gdh1995** [comment]
When this issue happens, can you use a real mouse to click a link? And a snapshot of `console` in "Developer Tools" may be helpful.

![image](https://user-images.githubusercontent.com/5547703/73335351-0ee53f80-42aa-11ea-8858-0f4c3dc1dd21.png)

**Comment by mattfloat** [comment]
I'm experiencing the same issue (and have been for months). Doesn't occur until I've used the browser for a bit (not sure if it depends on time or resource consumption) and keeps occurring until I restart the browser or the add-on.

I've recorded a minimal demo of the glitch. Note that the mouse pointer changes to its intended style (hand for link, I-beam for text) briefly every time the pointer moves across the respective area. Similarly, hovering a navbar on a "real" website then results in the nav items opening and collapsing for every pixel the mouse moves. 
Neither the tab's console nor the parent process browser console (not captured on video) threw any events during the demo.

https://github.com/philc/vimium/assets/2395971/47ed615b-d02a-48a6-b6a7-86ea340f900a

OS: macOS Sonoma, v14.5
Browser: Firefox Nightly 129.0a1 (2024-07-01) (64-bit)
Vimium version: 2.1.2

Hope this helps. Let me know if there's anything else you'd like me to check, @gdh1995

**Comment by gdh1995** [comment]
No idea... Sorry.
